stdcommands/forex/forex.go

```diff
@@ -39,7 +42,14 @@ var Command = &commands.YAGCommand{
 		from := check.Symbols[strings.ToUpper(data.Args[1].Str())]
 		to := check.Symbols[strings.ToUpper(data.Args[2].Str())]
 		if (to == nil) || (from == nil) {
-			return "Invalid currency code.\nCheck out available codes on: <https://api.exchangerate.host/symbols>", nil
+			_, err = paginatedmessages.CreatePaginatedMessage(
+				data.GuildData.GS.ID, data.ChannelID, 1, int(math.Ceil(float64(len(check.Symbols))/float64(15))), func(p *paginatedmessages.PaginatedMessage, page int) (*discordgo.MessageEmbed, error) {
```

Comment:
save this to a variable, which explains what it does. 

int(math.Ceil(float64(len(check.Symbols))/float64(15)))